remoteworkitem/github_test.go

```diff
@@ -66,7 +66,7 @@ func TestGithubFetchWithRateLimit(t *testing.T) {
 
 func TestGithubFetchWithRecording(t *testing.T) {
 	resource.Require(t, resource.UnitTest)
-	r, err := recorder.New("github")
+	r, err := recorder.New("../test/data/github")
```

Comment:
Could you make the recorded file name a little bit more specific to the test. 'github' is a bit generic.